packages/harpstrata/src/harpstrata/IsActive/IsActiveMatrix/types/index.ts

```diff
@@ -1,7 +0,0 @@
-export enum IsActiveIds {
-  Active = 'ACTIVE',
-  Inactive = 'INACTIVE',
-}
-
-export type IsActiveRow = ReadonlyArray<IsActiveIds | undefined>
-export type IsActiveMatrix = ReadonlyArray<IsActiveRow>
```

Comment:
What's still going on in this file?

Reply:
Looks like IsActiveMatrix has been completely removed. I don't know why it's being presented like this.